rest-service/manager_rest/upload_manager.py

```diff
@@ -659,9 +660,26 @@ def _prepare_and_process_doc(self,
                     '{version}'.format(archive_name=new_plugin.archive_name,
                                        package_name=new_plugin.package_name,
                                        version=new_plugin.package_version))
+            if self._is_plugin_installing(new_plugin, plugin):
+                raise manager_exceptions.ConflictError(
+                    'a plugin archive by the name of {archive_name} for '
```

Comment:
I will improve the existing wording by
                   '"{archive_name}" plugin archive for'
                    ' "{package_name}"package and  {version} version '
                    'is already being installed'